Example/Example/ViewController.swift

```diff
@@ -33,13 +41,21 @@ class ViewController: UITableViewController
         static let defaultButtonTitle =  "Go Ahead"
         static let cancelButtonTitle = "Cancel"
         static let moreInfoButtonTitle = "More Information"
+        
+        static let switchText = "Do not show this again"
     }
     
     // MARK: Fancy Alert
     
-    func showFancyAlert() {
+    func showFancyAlert(
+        moreInfoButton: FancyAlertViewController.Config.ButtonConfig? = nil,
```

Comment:
Is this extra space on purpose? Not sure if this is a new guideline, but it's a bit confusing!!